pom.xml

```diff
@@ -0,0 +1,208 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>Services</groupId>
+  <artifactId>ServicesUtilities</artifactId>
```

Comment:
We've gone the route on the Services team of using hyphenated names: https://github.com/search?q=topic%3Aservices-team+org%3AUCLALibrary&type=Repositories -- I do notice this hasn't been a consistent rule across all the library's repos though. What do you think about naming the artifact and the project `services-utilities`?